spark/src/test/java/com/netflix/iceberg/spark/data/TestParquetAvroReader.java

```diff
@@ -101,10 +99,11 @@ public void testStructSchema() throws IOException {
       // clean up as much memory as possible to avoid a large GC during the timed run
       System.gc();
 
-      try (ParquetReader<Record> reader = new ParquetReader<>(
-          Files.localInput(testFile), structSchema, ParquetReadOptions.builder().build(),
-          fileSchema -> ParquetAvroValueReaders.buildReader(structSchema, readSchema),
-          Expressions.alwaysTrue(), true)) {
+      try (CloseableIterable<Record> reader = Parquet.read(Files.localInput(testFile))
+          .project(structSchema)
+          .createReaderFunc(
+              fileSchema -> ParquetAvroValueReaders.buildReader(structSchema, readSchema))
+           .build()) {
```

Comment:
These changes to `TestParquetAvroReader.java` seem unrelated to PR intention. Can you explain?

Reply:
Rather than updating the ParquetReader call to include the new boolean for case sensitivity, I moved them to use the builder instead. It fixes the problem this time around, and makes sure it won't fail other tests in the future.